bluesky_queueserver/manager.py

```diff
@@ -387,10 +474,19 @@ async def zmq_server_comm(self):
             logger.info(f"ZeroMQ server sending response: {msg_out}")
             await self._zmq_send(msg_out)
 
+    # ------------------------------------------------------------
 
-if __name__ == "__main__":
-    re_manager = RunEngineManager()
-    try:
-        asyncio.run(re_manager.zmq_server_comm())
-    except KeyboardInterrupt:
-        logger.info("The application was stopped")
+    def run(self):
+        """
+        Overrides the `run()` function of the `multiprocessing.Process` class. Called
+        by the `start` method.
+        """
+        logger.info("Starting RE Manager process")
+        try:
+            asyncio.run(self.zmq_server_comm())
+        except Exception as ex:
+            logger.info("Exiting RE Manager with exception %s" % str(ex))
```

Comment:
The `logger.exception` method includes the whole traceback below the message.

```suggestion
            logger.exception("Exiting RE Manager")
```

Reply:
Actually I didn't know about `logger.exception`. I will think about how to use it. Actually there is some work on logging that needs to be done.